package.json

```diff
@@ -23,7 +23,7 @@
     "ember-cli-babel": "4.1.0",
     "ember-cli-htmlbars": "^0.7.4",
     "ember-cli-version-checker": "^1.0.2",
-    "velocity-animate": ">= 0.11.8"
+    "velocity-animate": "git+https://github.com/nonmanifold/velocity"
```

Comment:
Your velocity fork can't easily be compared to upstream because the diff is massive. Maybe whitespace problems?